recipes/xdotool/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "3.20211022.1" %}
+
+package:
+  name: xdotool
+  version: {{ version }}
+
+source:
+  url: https://github.com/jordansissel/xdotool/archive/v{{ version }}.tar.gz
+  sha256: 82b15a944a5e82fee15e0f6116bd9f642bc3d0bb6989fc0ca5ad9dfe35de0847
+
+build:
+  skip: true  # [not linux]
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - make
+    - pkg-config
+  host:
+    - xorg-libx11
+    - xorg-libxi
+    - xorg-libxtst
+    - xorg-libxinerama
+    - libxkbcommon
+  run:
+    - xorg-libxtst
```

Comment:
Should we add a run export here?

Reply:
I already have the PR. @hmaarrfk 
https://github.com/conda-forge/xorg-libxtst-feedstock/pull/9